scripts/build.js

```diff
@@ -10,6 +10,9 @@ const buildConfig = {
     "grid",
     "icon",
     "introduction",
+    "input-group",
+    "input-addon",
+    "button-input-addon",
```

Comment:
Heeft `button-input-addon` zelf een stylesheet nodig? in de docs staat dat die die van `input-addon` gebruikt.

Reply:
Klopt, das idd een subcomponent. Aangepast!.